logging/cluster-logging-external.adoc

```diff
@@ -41,7 +41,133 @@ include::modules/cluster-logging-collector-log-forward-syslog.adoc[leveloffset=+
 
 include::modules/cluster-logging-collector-log-forward-cloudwatch.adoc[leveloffset=+1]
 
+=== Forwarding logs to Amazon CloudWatch from STS enabled clusters
+
+For clusters with AWS Security Token Service (STS) enabled, you can create an AWS service account manually or create a credentials request using the xref:../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc[Cloud Credential Operator(CCO)] utility `ccoctl`.
+
+.Creating an AWS credentials request
+. Create a `CredentialsRequest` Custom Resource YAML using the template below:
++
+.CloudWatch Credentials Request Template
+[source,yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: <your_role_name>-credrequest
+  namespace: openshift-logging
```

Comment:
The default namespaces should be openshift-cloud-credential-operator rather than openshift-logging